Dockerfile

```diff
@@ -1,74 +1,71 @@
 FROM ubuntu:14.04
```

Comment:
maybe move to [Debian:8.3](https://imagelayers.io/?images=ubuntu:14.04,debian:8.3) can help